install/kubernetes/cilium/values.yaml

```diff
@@ -1,5 +1,143 @@
+<<<<<<< HEAD
```

Comment:
Looks like this file still has unresolved merge conflicts.

Reply:
Thanks, fixed.